src/components/stepFour/index.js

```diff
@@ -267,7 +267,7 @@ export class stepFour extends stepTwo {
   //FlatPricing
   getPricingStrategyParams = (web3, pricingStrategy, i, token) => {
     console.log(pricingStrategy);
-    let oneTokenInETH = roundToDecimals(1000000000000000000, 1/pricingStrategy.rate)
+    let oneTokenInETH = roundToDecimals(-18, 1/pricingStrategy.rate)
```

Comment:
what is -18? We should not have plain strings or numbers, these values should all be extracted to constants and imported. I would also make '1/pricingStrategy.rate' a variable.